scripts/bqeth_subscription.py

```diff
@@ -0,0 +1,171 @@
+import json
+import os
+
+import requests
+from dotenv import load_dotenv
+from web3 import Web3
+from web3.middleware import geth_poa_middleware
+
+"""
+Read from a .env file
+"""
+load_dotenv()
+PROVIDER_URL = os.environ.get("PROVIDER_URL")
+PRIVATE_KEY = os.environ.get("PRIVATE_KEY")
+
+
+MAX_NODES = 10
+NUCYPHER_DOMAIN = "lynx"
+CHAIN = "80002"
+PORTER_ENDPOINT = f"https://porter-{NUCYPHER_DOMAIN}.nucypher.community"
+
+with open(f"deployment/artifacts/{NUCYPHER_DOMAIN}.json", "r") as f:
+    registry = json.load(f)
+
+SUBSCRIPTION_CONTRACT_ADDRESS = registry[CHAIN]["BqETHSubscription"]["address"]
+SUBSCRIPTION_CONTRACT_ABI = registry[CHAIN]["BqETHSubscription"]["abi"]
+ERC20_CONTRACT_ADDRESS = registry[CHAIN][NUCYPHER_DOMAIN.title() + "RitualToken"]["address"]
+ERC20_CONTRACT_ABI = registry[CHAIN][NUCYPHER_DOMAIN.title() + "RitualToken"]["abi"]
+COORDINATOR_CONTRACT_ADDRESS = registry[CHAIN]["Coordinator"]["address"]
+COORDINATOR_CONTRACT_ABI = registry[CHAIN]["Coordinator"]["abi"]
+GLOBAL_ALLOW_LIST_CONTRACT_ADDRESS = registry[CHAIN]["GlobalAllowList"]["address"]
+GLOBAL_ALLOW_LIST_CONTRACT_ABI = registry[CHAIN]["GlobalAllowList"]["abi"]
```

Comment:
I see that you kept this as a pure web3.py script, but if you want to introduce ape functionality and custom utilities, there's a utility in `deployments.registry` that will clean this up a bit.

```
from deployment.registry import contracts_from_registry

contracts = contracts_from_registry('deployment/artifacts/lynx.json', 80002)
...
coordinator = contracts["Coordinator"]
...
coordinator.address
```

Reply:
👍🏻 What's the reason for that?

Reply:
when David and I chatted about it we agreed that keeping it as vanilla as possible was a good idea. It's to act as a demo for adopters, we don't want to force them into using `ape`